src/clear/sql/sql.cr

```diff
@@ -68,7 +72,15 @@ module Clear
     end
 
     def init(url : String)
-      @@connection = DB.open(url)
+      @@connections ||= {} of Symbolic => DB::Database?
+      @@connections.not_nil!["default"] = DB.open(url)
+    end
+
+    def init(connections : Hash(Symbolic, String))
+      @@connections ||= {} of Symbolic => DB::Database?
```

Comment:
it would be great to have the method able to attach another connections every time it's executed:

```crystal
Crystal::SQL.init("default", "postgres://xxxxx")
Crystal::SQL.init("secondary", "postgres://xxxxx")
```